src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js

```diff
@@ -1484,7 +1557,7 @@ export default compose(
 
         const valuesForPlot =
           variable.plotTypes === 'categorical'
-            ? [...valuesForTable]
+            ? [...filteredData.map(d => d.keyArray).slice(0, 2)]
```

Comment:
this makes survival plots work for custom categorical buckets - it sends in arrays of real property names (e.g. `[male, female, not reported]`) instead of the custom bucket name.